PWGGA/GammaConvBase/AliConversionMesonCuts.cxx

```diff
@@ -1784,7 +1788,7 @@ Bool_t AliConversionMesonCuts::SetCut(cutIds cutID, const Int_t value) {
         UpdateCutString();
         return kTRUE;
       } else return kFALSE;
-    } else if(fUsePtDepSelectionWindow && fMesonKind == 2){
+    } else if(fUsePtDepSelectionWindow || fMesonKind == 3){
```

Comment:
Why was this changed from && to || ?
Now everyone that has fUsePtDepSelectionWindow set to true will enter the eta prime case.

Reply:
That's a very good point, I meant to delete the flag from there completely. Thank you!